drivers/bluetooth/hci/Kconfig.nxp

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright 2023 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config HCI_NXP_SET_BDADDR
+	bool "HCI Set BT Address"
+	help
+	  If enabled, the BD_ADDR will be set by the host at HCI init, using a HCI vendor command.
```

Comment:
This option doesn't seem to be used anywhere - maybe it should be removed?

Reply:
Indeed, this isn't used for now. Removed